README.md

```diff
@@ -14,27 +14,24 @@ and add to your path.
 
 # Usage 
 
-Before usage, you should initialize. It sets up your github integration with your developer environments.
-It requires the following:
-- initialization file: [init-example.yaml](examples/init-example.yaml)
-- your person GitHub token to perform operations on your behalf. See more info [here](#GitHub-Access-Token)
+## Initialisztion
```

Comment:
typo

Reply:
Also perhaps we should leave it as is and say it requires either an initialization file, OR an environments repo that contains it.

Reply:
Fixed, thanks for spotting that!